application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -204,7 +206,10 @@ private RestAction<Message> dmUser(MessageChannelUnion sourceChannel, ForumPost
     }
 
     private RestAction<Void> deleteOriginalMessage(JDA jda, String channelId, String messageId) {
-        return jda.getTextChannelById(channelId).deleteMessageById(messageId);
+        TextChannel sourceChannel = Objects.requireNonNull(jda.getTextChannelById(channelId),
+                "Source channel could not be found for transfer-question feature");
+
+        return sourceChannel.deleteMessageById(messageId);
```

Comment:
what was the issue with the previous implementation?

Reply:
It's nothing too scary, possibly null value in case there's no such ID or if cache isn't initialized(latter isn't the problem i think, zabu mentioned something along the lines). IDE was screaming at me, so just to be on the safer side i chose to change this.